.gitignore

```diff
@@ -104,3 +104,6 @@ ENV/
 .ropeproject
 
 .idea/
+
+# Api keys
+apikey.json
```

Comment:
This pull request should only contain bdd.yml file. Remaining changes should not be a part of this ticket. Please remove them.